acquia/mc-cs-plugin-custom-objects
# Custom objects: Reports

This PR implements ability to use Custom Objects as the data source in Reports.
 https://backlog.acquia.com/browse/MAUT-994

Steps to test:
1. Create a custom object that uses all available types of fields (`Custom Object #1`).
2. Create multiple instances of `Custom Object #1`.
3. Associate instances of `Custom Object #1` with some leads and companies.
4. Create a report based on the custom object. Make sure that the data you see in the report is correct.
5. Try to use filters and grouping in your report. Try it with different fields. Try to apply date filter.
6. Pay special attention to the "Multiselect" field type when you test.
7. Make sure that the data you see in the report is correct.
8. Create another type of custom object (`Custom Object #2`). Make sure you are able to select it from the list of the data sources in the Reports. You don't have to use all the fields, I think 2-3 fields are enough.
9. Create multiple instances of `Custom Object #2`.
10. Associate instances of `Custom Object #2` with some leads and companies.
11. Create a report based on `Custom Object #2`. Make sure that the data you see in the report is correct and it is not mixed with the data from the first report.
